app/components/Dashboard/PriceHistoryPanel/PriceHistoryPanel.jsx

```diff
@@ -40,11 +46,17 @@ type Props = {
 }
 
 const DURATIONS: { [key: Duration]: string } = {
-  '1m': '1 month',
+  '1d': '1 day',
   '1w': '1 week',
-  '1d': '1 day'
+  '1m': '1 month'
```

Comment:
Since #1626 requests the default duration be 1 day, I assume this should be the dropdown option order.

Reply:
Sidenote: afaik the order of these entries is not guaranteed. If this is an issue, we can either substitute for an Array (which required some code adjustments I would rather avoid) or perhaps utilize Immutable's `OrderedMap`.

Reply:
The new js Map is ordered as well (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) no need for immutable OrderedMap

Reply:
Awesome! Ok that it's supported in IE11 and up?

Reply:
Oh right, it's all node based.. takes getting used to after a lifetime of browser compatibility considerations 😅